tests/test_euclidean_space.py

```diff
@@ -22,83 +23,13 @@ def setUp(self):
 
         self.n_samples = 10
 
-    def test_belongs(self):
-        point = self.space.random_uniform()
-        belongs = self.space.belongs(point)
-        expected = gs.array([[True]])
-
-        gs.testing.assert_allclose(belongs, expected)
-
-    def test_random_uniform(self):
-        point = self.space.random_uniform()
-
-        gs.testing.assert_allclose(point.shape, (1, self.dimension))
-
+    @geomstats.tests.np_only
```

Comment:
any reason to start with the @geomstats.tests.np_only tests? It makes it harder to see in the PR if no test has been deleted.